export_node.lua

```diff
@@ -8,11 +8,7 @@ for k, v in pairs(process) do
     if v.args then
         for i, vv in pairs(v.args) do
             if #vv > 0 then
-                v.args[i] = {
-                    name = vv[1],
-                    type = vv[2],
-                    desc = vv[3],
-                }
+                v.args[i] = vv

```

Comment:
This is not an array this is a tablehttps://github.com/zhandouxiaojiji/behavior3lua/blob/master/behavior3/nodes/conditions/cmp.lua#L14-L21